elm.json

```diff
@@ -5,7 +5,8 @@
     "license": "BSD-3-Clause",
     "version": "1.0.1",
     "exposed-modules": [
-        "RadixTree"
+        "RadixTree",
+        "UnOrderedRadixTree"
```

Comment:
`UnorderedRadixTree`? Sorry, I don't know if I'm annoying with this :cold_sweat:

Reply:
No worries.